pkg/command/create/config/example.toml

```diff
@@ -0,0 +1,70 @@
+[global]
```

Comment:
It's a little bit weird to use this config instead of YAML

Reply:
IMO, if users use the YAML, they can use the `kubectl apply` directly

Reply:
The configuration file just provides a simple way for the user to create a specific instance, if the user needs a more complex configuration they can just use YAML. And Toml file is simple enough and easy to use.

Reply:
okay, sounds good.